infrastructure/cdk/bin/index.ts

```diff
@@ -1,14 +1,22 @@
 #!/usr/bin/env node
 import 'source-map-support/register'
 import * as cdk from 'aws-cdk-lib'
+import { pascalCase } from '@casimir/lib'
```

Comment:
How long did it take to familiarize yourself with the SAM/CDK? This all looks great.